### PR TITLE
Adds atomic-helper spec

### PR DESCRIPTION
**Pull request by anselmbradford**
## Additions

- Adds spec for atomic-helpers utility method.

## Testing

- `gulp test:unit:scripts` should pass.

## Review

- @jimmynotjim 
- @sebworks 
- @KimberlyMunoz

### CODE REVIEW

File: test/unit_tests/modules/util/atomic-helpers-spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var assert = chai.assert;
```
Comment (KimberlyMunoz) [inline test/unit_tests/modules/util/atomic-helpers-spec.js]: Could we stick with expect/BDD. There should be a way to use .throw with the expect syntax.

http://chaijs.com/api/bdd/#method_throw

Comment (anselmbradford) [inline test/unit_tests/modules/util/atomic-helpers-spec.js]: Ahh sure! What's the difference between using expect and assert? Just a style?

Comment (KimberlyMunoz) [inline test/unit_tests/modules/util/atomic-helpers-spec.js]: Yeah, it's mostly the syntax for how you write the tests. Expect is a style for behavior driven development, Assert is a style more for test driven development. There's a good explanation of the differences here. 

http://chaijs.com/guide/styles/

Comment (anselmbradford) [inline test/unit_tests/modules/util/atomic-helpers-spec.js]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/1701/commits/6c4e6138b154fa642dab80dd973d720ca1690b5b

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by sebworks** [comment]
:+1:

**Pull request merged**